MapboxNavigation/NavigationMapView.swift

```diff
@@ -262,6 +266,8 @@ open class NavigationMapView: MGLMapView, UIGestureRecognizerDelegate {
         addGestureRecognizer(mapTapGesture)
         
         installUserCourseView()
+        
+        styleURL = MGLStyle.navigationDayStyleURL
```

Comment:
This is worth a changelog blurb that NavigationMapView now uses `MGLStyle.navigationDayStyleURL` by default.

Reply:
Will skip adding `MGLStyle.navigationDayStyleURL` in this PR to prevent and regression as per this [comment](https://github.com/mapbox/mapbox-navigation-ios/pull/2535#discussion_r470764973). Separate ticket will be created to tackle this.